parts/linux/cloud-init/artifacts/manifest.json

```diff
@@ -44,7 +44,7 @@
             "1.22.15",
             "1.23.8-hotfix.20220620",
             "1.23.12",
-            "1.23.15",
+            "1.23.15-v1.23.15-hotfix.20230114",
```

Comment:
typo which led to the VHD failures: `1.23.15` repeated twice

Reply:
Fixed now